tools/license

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+# inspired by http://stackoverflow.com/a/151699/5031322
+license_length=$(wc -l LICENSE | cut -f1 -d ' ')
+files="${*:-$(git ls-files 'tests/*.pm')}"
+for x in $files; do
+    tmp=$(mktemp)
+    head -$license_length $x | diff LICENSE - || ( (cat LICENSE; echo; cat $x) > $tmp; mv $tmp $x)
```

Comment:
the `||` if I understand correctly will not make it fail the CI if someone pushes some file with incorrect license (because it will replaced only in the container not in the pull request) but it is very useful for passing the CI in your local env before pushing. We might want to replace all header and fail as well, WDYT?